tests/telemetry/test_telemetry.py

```diff
@@ -464,6 +465,89 @@ def test_validate_entries(monkeypatch):
     assert res == (event_id, uid, action, client_time, elapsed_time)
 
 
+def test_get_latest_version(monkeypatch):
+    is_latest = telemetry.get_latest_version()
+    assert isinstance(is_latest, str)
+    version_index = [i for i, ltr in enumerate(is_latest) if ltr == '.']
+    assert len(version_index) >= 1
+
+    mock_httplib = Mock()
+    mock_httplib.HTTPSConnection().request.side_effect = Exception
+    monkeypatch.setattr(telemetry, 'httplib', mock_httplib)
+    is_latest = telemetry.get_latest_version()
+    assert is_latest == ploomber.__version__
+
+    # Mock version and the conf, check it produces the same version
+
+
+def write_to_conf_file(tmp_directory, monkeypatch, last_check):
+    stats = Path('stats')
+    stats.mkdir()
+    path = stats / 'config.yaml'
+    monkeypatch.setattr(telemetry, 'DEFAULT_HOME_DIR', '.')
+    (path).write_text("stats_enabled: True\n"
+                      f"version_check_enabled: {last_check}\n")
+
+
+def test_version_skips_when_updated(tmp_directory, capsys, monkeypatch):
+    # Path conf file
+    monkeypatch.setattr(telemetry, '__version__', '0.14.8')
```

Comment:
FYI @idomic : i fixed this test, instead of patching `ploomber.__version__` you need to patch the local copy, `telemetry.__version__`